packages/amplify-migration-tests/package.json

```diff
@@ -18,10 +18,11 @@
   "private": true,
   "scripts": {
     "build-tests": "tsc --build tsconfig.tests.json",
-    "migration_v4.28.2_nonmultienv_layers": "npm run setup-profile 4.28.2 && jest src/__tests__/migration_tests/lambda-layer-migration/layer-migration.test.ts --verbose --passWithNoTests",
-    "migration_v4.52.0_multienv_layers": "npm run setup-profile 4.52.0 && jest src/__tests__/migration_tests/lambda-layer-migration/layer-migration.test.ts --verbose --passWithNoTests",
-    "migration_v5.2.0": "npm run setup-profile 5.2.0 && jest --testPathIgnorePatterns=auth.deployment.secrets\\|layer-migration\\|overrides --verbose --passWithNoTests",
-    "migration_v6.1.0": "npm run setup-profile 6.1.0 && jest --testPathIgnorePatterns=auth.deployment.secrets\\|layer-migration --verbose --passWithNoTests",
+    "migration_v4.28.2_nonmultienv_layers": "npm run setup-profile 4.28.2 && jest src/__tests__/migration_tests/lambda-layer-migration/layer-migration.test.ts --verbose",
+    "migration_v4.52.0_multienv_layers": "npm run setup-profile 4.52.0 && jest src/__tests__/migration_tests/lambda-layer-migration/layer-migration.test.ts --verbose",
+    "migration_v5.2.0": "npm run setup-profile 5.2.0 && jest --verbose",
+    "migration_v6.1.0": "npm run setup-profile 6.1.0 && jest --verbose",
+    "migration_v10.5.1": "npm run setup-profile 10.5.1 && jest --verbose",
```

Comment:
previously, all of these JEST commands were using regex to exclude tests, this is the wrong way to do it, as we were still creating jobs for these tests but Jest would skip the tests and mark the jobs as passing. 

I moved this logic to our split-e2e-test script. Now, jobs are only created if the test should exist.

Reply:
should we think about failing a test run if zero tests have been found?

Reply:
Yeah ; I did this already, previously it said "--allow pass if no tests found" , thats been removed